packages/components/src/DateTimePickers/DatePicker/CalendarHeader/CalendarHeader.scss

```diff
@@ -0,0 +1,11 @@
+
+.container {
+    display: flex;
+    direction: column;
```

Comment:
`flex-direction`
But here it is a mistake, they are in row direction. Remove that please

Reply:
&#10003;